Sources/MongoSwift/MongoSwift.stencil

```diff
@@ -0,0 +1,30 @@
+// swiftlint:disable:previous vertical_whitespace
```

Comment:
Two thoughts:

- could we just have a `MongoSwiftVersion.swift` (or `Version.swift`) that adds a `private static let versionString` to the right namespace. It seems like there is other stuff going on in this file that we don't want to be templated
- somewhat crazier, but could be nice, can you encode the version into the `Package.swift` and dynamically read it from that file at runtime? `Package.swift` will always be there, in a known fixed location

Reply:
I went with the first!